src/components/IconButton/IconButton.modules.scss

```diff
@@ -1,3 +1,9 @@
-.referenceContainer {
+.wrapper {
+  width: fit-content;
```

Comment:
please ask Orr if it is ok to use this property (fit-content) because it's looks like it not supported in all browsers: https://caniuse.com/?search=fit-content